### ISSUE TITLE
[Enhancement]: Figma library for Carbon Charts

### ISSUE THREAD

**Issue by mzuliani-ibm** [carbon-design-system/carbon-charts#1392]
### Contact Details

zuliani@ca.ibm.com

### Summary

Would like something equivalent to the Carbon Charts Sketch library available for designers working in Figma.

### Justification

Many designers are working natively in Figma and need to be able to include Carbon Charts in their designs and prototypes.

### Desired UX and success metrics

A designer can quickly mock-up a Carbon UI and include Carbon Charts without importing from the existing Carbon Charts Sketch library.

### "Must have" functionality

Figma library

### Specific timeline issues / requests

Team is moving to Figma now, so the sooner the better.

### Available extra resources

Visual and UX designers on the team might be able to help.

**Comment by jeanservaas** [comment]
@mzuliani-ibm 

Yes totally agreed, we'd love to have a Figma kit for charts but it would definitely require help from the community to make this happen.

Right now there is a Figma guild that’s helping to migrate sketch kits, but
 
1) they're already swamped with other migrations. For instance, the carbon for ibm.com library hasn't been completed yet and the mobile kit hasn't even been started. Unfortunately these are months long efforts involving a visual lead on Carbon and 10-12 volunteers.

2) part of the problem with charts is that we don’t even have a symbolized Sketch kit for charts, just a file that people can pull from. So there would need to be a deeper level of effort to figure out how to symbolize the charting elements and create a proper library that we can push updates too.

Supporting chart users is always top of mind, but I don't see this happening in the near future unless someone from the community steps up to spearhead the effort.
